src/Components/ReturnForm/style.css

```
@@ -1,3 +1,8 @@
+#root__title {
+  font-size: 200%;
```

I think this would be better written as
```suggestion
  font-size: 2em;
```